src/VisualStudio/IntegrationTest/TestUtilities/Common/ProjectUtilities.cs

```diff
@@ -9,10 +9,13 @@ public abstract class Identity
 
     public class Project : Identity
     {
-        public Project(string name)
+        public Project(string name, string relativePath = null)
         {
             Name = name;
+            RelativePath = relativePath;
         }
+
+        public string RelativePath { get; }
```

Comment:
But most of the time won't the `RelativePath` simply be the `Name` plus .csproj or .vbproj?